ui/design-system-core/src/components/ActionDropDown/index.tsx

```diff
@@ -8,28 +8,31 @@ import { IconType } from '@akashaorg/typings/ui';
 import { TextProps } from '../Text';
 import { getColorClasses } from '../../utils/getColorClasses';
 
-type Action = {
+export type Action = {
   label: string;
-  icon: IconType;
-  color?: TextProps['color'];
-  onClick: (event: React.SyntheticEvent<Element, Event>) => void;
-};
+  icon?: IconType;
+  onClick: (index: number) => void;
+} & TextProps;
 
 export type ActionDropdownProps = {
   actions: Action[];
   customStyle?: string;
+  showBorder?: boolean;
   ref?: LegacyRef<HTMLDivElement>;
 };
 
 const ActionDropdown: React.FC<ActionDropdownProps> = forwardRef(
```

Comment:
Hey @didd , can you add a some docs to this component? A small description perhaps because from the name it's not immediately clear.

Reply:
hey Sever, done. I have renamed the prop as well.

Reply:
oh, my bad, I was thinking to some docs regarding the ActionDropDown component, it's a dropdown with some actions inside?

Reply:
oh, okay, will do. I saw the highlight on the prop and I thought you meant that.

Reply:
I have renamed into a `List` component. Initially it was created to cater for action dropdown shown below.
![image](https://user-images.githubusercontent.com/35398733/232745462-8b095a5f-b59d-45a7-83d8-57c33445ee3a.png)

However, we have use case for it on auto complete component as well. I think the name list suits it better.

![image](https://user-images.githubusercontent.com/35398733/232745282-a698365a-5618-49d9-b35c-4e21e13091e7.png)

Reply:
yes, much better